data-reconciliation-app/test/unit-test/services/ingest-service.test.ts

```diff
@@ -13,30 +12,42 @@ describe("Data Ingestion Service", () => {
   afterEach(() => {});
 
   test("Should ingest data successfully", () => {
-    // // Act
-    // const testDataRecords: DataRecord[] = [
-    //   DataRecord.create({key:"1", value: "test1"}).content!,
-    //   DataRecord.create({key:"2", value: "test2"}).content!,
-    //   DataRecord.create({key:"3", value: "test3"}).content!,
-    // ];
-    // // Assert
-    // const result = ingestService.submitData(userId, testDataRecords);
-    // // Assert
-    // expect(result).not.toBeNull();
-    // expect(result.success).toBe(true);
-    // expect(result.content).toBe("data has ingested successfully");
+    // Act
+    const testDataRecords: DataRecord[] = [
+      DataRecord.create({ key: "1", value: "test1" }).content!,
+      DataRecord.create({ key: "2", value: "test2" }).content!,
+      DataRecord.create({ key: "3", value: "test3" }).content!,
+    ];
+    // Assert
+    const result = ingestService.submitData(userId, testDataRecords);
+    // Assert
+    expect(result).not.toBeNull();
+    expect(result.success).toBe(true);
+    expect(result.content).toBe("data has ingested successfully");
+  });
+
+  test("Should ingest empty data successfully", () => {
+    // Act
+    const testDataRecords: DataRecord[] = [];
+
+    // Assert
+    const result = ingestService.submitData(userId, testDataRecords);
+
+    // Assert
+    expect(result).not.toBeNull();
+    expect(result.success).toBe(true);
+    expect(result.content).toBe("data has ingested successfully");
```

Comment:
Should empty data be allowed? IMHO we could block this and treat similarly to `null` data